molgenis-data/src/main/java/org/molgenis/data/support/DefaultEntity.java

```diff
@@ -275,7 +276,15 @@ public Entity getEntity(String attributeName)
 		EntityMetaData ref = attribute.getRefEntity();
 		ids = ids.stream().map(attribute.getDataType()::convert).collect(Collectors.toList());
 
-		return dataService.findAll(ref.getName(), IN(ref.getIdAttribute().getName(), ids));
+		String entityIdName = ref.getIdAttribute().getName();
+		Map<Object, Entity> entities = new HashMap<Object, Entity>();
```

Comment:
In MySql, when selecting the row, we've already retrieved lists of IDs (and up to this pull request used to trunkate the string containing the ID list to 1024 chars). In Elasticsearch, we store the full list of children in a document. So is this an academic point or do we actually also have a backend that wouldn't fail elsewhere for MREF lists in the hundreds of MB size of magnitude?